sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_constants.hpp

```diff
@@ -0,0 +1,12 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+namespace Azure { namespace Storage { namespace Files { namespace DataLake { namespace Details {
+  // Error codes:
+  constexpr static const char* DataLakeFilesystemNotFound = "FilesystemNotFound";
+  constexpr static const char* DataLakePathNotFound = "PathNotFound";
```

Comment:
I think if you define error code as constant strings, you should also define all other error codes here.

Reply:
It's in the `Details` namespace. I don't think there is a hard requirement on defining them all.

Reply:
Well, I don't feel strong about it. So suit yourself